LongWarOfTheChosen/Config/XComAI.ini

```diff
@@ -78,8 +78,17 @@
 +NewBehaviors=(BehaviorName=GenericAIRoot, NodeType=Selector, Child[0]=TryNonAggressiveBehavior, Child[1]=TryMindControlledRoot, Child[2]="::CharacterRoot", Child[3]=RandOverwatch50, Child[4]=DoCower, Child[5]=SkipMove)
 
 ; "GenericScamperRoot" - If the unit is unable to find a valid scamper tile, try again with hazards turned off.
+; Naja scamper enabled via ini, check added on Child 1 of GenericScamperRoot. Everything new behavior except ScamperRoot_Naja can be nuked after uc implementation.
 +BehaviorRemovals="GenericScamperRoot"
-+NewBehaviors=(BehaviorName=GenericScamperRoot, NodeType=Selector,       Child[0]=CounterBeaglerushManeuver,       Child[1]=MoveStandardWithLoSUnsafe,       Child[2]=MoveStandardUnsafe, Child[3]=MoveStandardWithLoSUnsafeIgnoreHazards, Child[4]=MoveStandardUnsafeIgnoreHazards, Child[5]=SkipMove)
++NewBehaviors=(BehaviorName=GenericScamperRoot, NodeType=Selector, Child[0]=CounterBeaglerushManeuver, Child[1]=NajaScamperCheck, Child[2]=MoveStandardWithLoSUnsafe, Child[3]=MoveStandardUnsafe, Child[4]=MoveStandardWithLoSUnsafeIgnoreHazards, Child[5]=MoveStandardUnsafeIgnoreHazards, Child[6]=SkipMove)
```

Comment:
That is absolutely not the way to do scamper. Instead, change the unit template's `CharTemplate.strScamperBT`

Reply:
Yeah, that's personal code I've been using since I can't compile .uc, probably was not a good idea to add it to pull request. I've removed the change and only left ScamperRoot_Naja behaviour.